modules/aws-eks-managed-node-groups/iam.tf

```diff
@@ -1,4 +1,6 @@
 resource "aws_iam_role" "managed_ng" {
+  count = local.create_iam_role_count
```

Comment:
just FYI - this is now a breaking change

Reply:
I am adding this for users who want to bring their own IAM role to node groups. May be we can advise on mv state file as you recommended. I think its better to do it now rather than later